plugins/LilyPad/resource.h

```diff
@@ -46,7 +46,7 @@
 #define IDC_PAD_LIST                    1130
 #define IDC_COMBO1                      1131
 #define IDC_PAD_TYPE                    1132
-#define IDC_SLIDER1                     0x1000
+#define IDC_SLIDER_SENSITIVITY1         0x1000
```

Comment:
There's definitely a theme to it as there are others (IDC_AXIS_DEVICE1, IDC_FLIP1), I assume it's  just there because there was once more than one set for the different pads or analog sticks. 

I can remove it, but then I'd have to remove it from the others as well.